Tribler/Policies/BoostingSource.py

```diff
@@ -287,7 +281,36 @@ def _on_database_updated(self, dummy_subject, dummy_change_type, dummy_infohash)
         self.database_updated = True
 
     def get_source_text(self):
-        return self.channel.name if self.channel else None
+        return str(self.channel_dict[2]) if self.channel_dict else None
+
+    def _load_torrent(self, infohash):
+        """
+        function to load infohash and put in into database.
```

Comment:
If I understand the method correctly, this method will start a download from a given infohash right? That's not in line with your comment.